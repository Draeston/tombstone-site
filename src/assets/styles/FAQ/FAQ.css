.faq {
    background: black;
    min-height: 700px;
    padding-bottom: 20px;
}
