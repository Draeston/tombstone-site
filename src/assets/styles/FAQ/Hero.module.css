@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Arabic:wght@200;300;400&display=swap');

.block {
    background: url("../../images/FAQ/faq-background.jpg") no-repeat;
    background-size: cover;
    background-position-y: 70%;
    height: 400px;
    position: relative;
}

.title {
    font-family: 'IBM Plex Sans Arabic', sans-serif;
    font-size: 48px;
    color: white;
    letter-spacing: 8px;

    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
}
