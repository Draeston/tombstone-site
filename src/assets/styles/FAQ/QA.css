@import url('https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@1,300;1,400&display=swap');

.qa-block {
    background: white;
    padding: 20px;
    width: 80%;
    margin: 25px auto;
    border-radius: 10px;
    box-shadow: 3px 3px 7px 5px rgba(255, 255, 255, 0.36);
}

.question {
    font-family: 'Ubuntu', sans-serif;
    font-weight: 400;
    color: black;
}

.answer {
    font-family: 'Ubuntu', sans-serif;
    font-weight: 300;
    color: black;
}
