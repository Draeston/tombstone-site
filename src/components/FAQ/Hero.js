import styles from "../../assets/styles/FAQ/Hero.module.css";

function Hero() {
  return (
    <div className={styles.block}>
      <h1 className={styles.title}>FAQ</h1>
    </div>
  );
}

export default Hero;
