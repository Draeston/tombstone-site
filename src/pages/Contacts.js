import { Card } from "../components/Contacts/Card";

export function Contacts(props) {
  const listItems = props.contacts.map((contact) => (
    <Card img={contact.img} name={contact.name} telegram={contact.telegram} />
  ));
  return <div>{listItems}</div>;
}
