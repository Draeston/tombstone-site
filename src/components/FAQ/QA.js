import "../../assets/styles/FAQ/QA.css";

export function QA(props) {
  return (
    <div className="qa-block">
      <p className="question">{props.question}</p>
      <p className="answer">{props.answer}</p>
    </div>
  );
}
