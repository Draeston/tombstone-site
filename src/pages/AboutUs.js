import {Hero} from "../components/AboutUs/Hero";
import {Info} from "../components/AboutUs/Info";

export function AboutUs () {
    return (
        <div>
            <Hero />
            <Info />
        </div>
    )
}
