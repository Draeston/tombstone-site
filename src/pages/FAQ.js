import "../assets/styles/FAQ/FAQ.css";
import { QA } from "../components/FAQ/QA";
import Hero from "../components/FAQ/Hero";

export function FAQ(props) {
  const listItems = props.qas.map((item) => (
    <QA question={item.question} answer={item.answer} key={item.question} />
  ));
  return (
    <div className="faq">
      <Hero />
      {listItems}
    </div>
  );
}
