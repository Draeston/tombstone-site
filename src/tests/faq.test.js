import React from "react";
import renderer from "react-test-renderer";
import { FAQ } from "../pages/FAQ";

test("snapshot faq", () => {
  const tree = renderer
    .create(<FAQ qas={[
        {
          question: "What are you doing here??",
          answer:
            "This site is designed to sell custom tombstone designs. " +
            "Just imagine how sad it would be if your tombstone will not show your artistic and aesthetic preferences. With " +
            "our site, you would not have to worry about all that stuff and may enjoy your afterlife with the tombstone of your choice.",
        },
    {
      question: "Is it a real tombstone?",
      answer:
      "Yes, it does not get more real than ours! Buying from us, you will get a first class marble tombstone.",
    },
    {
      question: "Can you deliver it to my place??",
      answer: "Yes, as long as you will pay for it!",
    },
    {
      question: "And what if I change my mind?",
      answer:
      "For that, you will get a 5% discount on a new tombstone design. However, we " +
      "are working on a subscription model where you could" +
      " pay every month and have your tombstone design ready for you to pass away.",
    },
      ]} />)
    .toJSON()
  expect(tree).toMatchSnapshot();
})
