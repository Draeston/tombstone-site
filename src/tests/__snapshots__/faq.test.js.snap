// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot faq 1`] = `
<div
  className="faq"
>
  <div
    className="block"
  >
    <h1
      className="title"
    >
      FAQ
    </h1>
  </div>
  <div
    className="qa-block"
  >
    <p
      className="question"
    >
      What are you doing here??
    </p>
    <p
      className="answer"
    >
      This site is designed to sell custom tombstone designs. Just imagine how sad it would be if your tombstone will not show your artistic and aesthetic preferences. With our site, you would not have to worry about all that stuff and may enjoy your afterlife with the tombstone of your choice.
    </p>
  </div>
  <div
    className="qa-block"
  >
    <p
      className="question"
    >
      Is it a real tombstone?
    </p>
    <p
      className="answer"
    >
      Yes, it does not get more real than ours! Buying from us, you will get a first class marble tombstone.
    </p>
  </div>
  <div
    className="qa-block"
  >
    <p
      className="question"
    >
      Can you deliver it to my place??
    </p>
    <p
      className="answer"
    >
      Yes, as long as you will pay for it!
    </p>
  </div>
  <div
    className="qa-block"
  >
    <p
      className="question"
    >
      And what if I change my mind?
    </p>
    <p
      className="answer"
    >
      For that, you will get a 5% discount on a new tombstone design. However, we are working on a subscription model where you could pay every month and have your tombstone design ready for you to pass away.
    </p>
  </div>
</div>
`;
